rudder-lang/src/parser.rs

```diff
@@ -978,12 +979,10 @@ pub struct PFile<'src> {
     pub code: Vec<PDeclaration<'src>>,
 }
 fn pfile(i: PInput) -> PResult<PFile> {
-    all_consuming(sequence!(
+    all_consuming(wsequence!(
```

Comment:
this is a sequence because the header is fixed and must not be prefixed by whitespace
This is strict to make sure the header can be parsed with other tools like shell

Reply:
fixed